docs/versioned/admin/admin-overview.md

```diff
@@ -0,0 +1,125 @@
+- Channels. See [Configure Channel defaults](/eventing/configuration/channel-configuration.md).
+
+### Blog articles
+
+| Article | Description |
+| --- | --- |
+| [Configurable RuntimeClassNames](../../blog/articles/kubevirt_meets_eventing.md) | Configure the default the RuntimeClassName field for deployments as selected from the Knative Service label selector. |
+
+## Monitoring and Observability
+
+Knative is instrumented with OpenTelemetry. For metrics, see the [Metrics Reference](/serving/observability/metrics/serving-metrics.md) and [Collecting metrics](/serving/observability/metrics/collecting-metrics.md). You can access [CloudEvents traces](/serving/observability/accessing-traces.md).
+
+For logging Serving activity, see [Collecting logs](/serving/observability/logging/collecting-logs.md), [Configuring logging](/serving/observability/logging/config-logging.md), and [Configuring Request logging](/serving/observability/logging/request-logging.md).
+
+For logging Eventing activity, see [Collecting logs](/eventing/observability/logging/collecting-logs.md) and [Configuring logging](/eventing/observability/logging/config-logging.md).
+
+### Blog articles
+
+| Article | Description |
+| --- | --- |
+| [Monitoring Virtual Machines with Knative Eventing](../../blog/articles/kubevirt_meets_eventing.md) | Monitor KubeVirt VMs with Knative Eventing building blocks. |
+| [Distributed tracing with Knative, OpenTelemetry and Jaeger](../../blog/articles/distributed-tracing.md) | Describes using Jaeger through a mix of OpenCensus and OpenTelemetry. |
+
+## Security and Access Control
+
+[Security-Guard](/serving/app-security/security-guard-about.md) provides visibility into the security status of applications deployed by Knative Services. See [Verifying Knative Images](/reference/security/verifying-images.md) for verifying binaries for authenticity before deploying.
+For access control, see [Transport Encryption](/eventing/features/transport-encryption.md), [Sender Identity](/eventing/features/sender-identity.md), and [cert-manager](/install/installing-cert-manager.md).
+
+### Blog articles
+
+| Article | Description |
+| --- | --- |
+| [Knative and Backstage](../../blog/articles/knative-backstage-plugins.md) | A platform for building developer portals.|
+| [Knative Backstage Security](../../blog/articles/knative-backstage-security.md) | How to secure the communication between the Knative Event Mesh plugin and the backend. |
+
+## Updates and Maintenance
+
+For Knative updates, see [About upgrading Knative](/install/upgrade/README.md) and [Checking your Knative version](/install/upgrade/check-install-version.md). The current Knative version is the shown in the version drop-down in the upper-left corner of this page. Refer to  [Upgrading with the Knative Operator](/install/upgrade/upgrade-installation-with-operator.md) or [Upgrading with kubectl](/install/upgrade/upgrade-installation.md).
```

Comment:
Change
"About upgrading Knative"
to
"Upgrading Knative"
(here and the actual linked heading).